packages/core/src/tag/Tag.css

```diff
@@ -25,7 +25,7 @@
 }
 .saltTag-secondary {
   background: var(--tag-secondary-background, var(--salt-category-1-bold-background));
-  color: var(--tag-secondary-foreground, var(--salt-category-1-bold-foreground));
+  color: var(--tag-secondary-foreground, var(--salt-content-bold-foreground));
```

Comment:
@joshwooding As this is defined at the top level, does it need to be repeated in each .saltTag-category-_n_ ? Just wondering if it can be defined once and save us the repetition.